wmn-data.json

```diff
@@ -1201,6 +1201,22 @@
         "known" : ["jaxjames", "jordan-2"],
         "cat" : "xx NSFW xx"
        },
+       {

```

Comment:
Please move this site above the `cowboys4angels` site since `cou` comes before `cow` when alphabetically sorted.